src/server_status.cc

```diff
@@ -0,0 +1,84 @@
+#include "server_status.h"
+
+#include <string>
+#include <vector>
+#include <map>
+
+namespace http{
+namespace server{
+
+
+
+long long
+ServerStatus::getRecordNum(){
+	return total_visit;
+}
+
+std::string
+ServerStatus::responseToString(ResponseCode response){
```

Comment:
Actually I am thinking of using string directly here instead of the Response structure. Let me make the change latter.